### PR TITLE
Release 4.3.1

### PR DESCRIPTION
**Pull request by ashie**
Fix ##368, #373

### CODE REVIEW

**Comment by ashie** [comment]
I'll wait Ruby 2.7.6: https://github.com/ruby/ruby/releases/tag/v2_7_6

**Comment by cosmo0920** [comment]
FYI: RubyInstaller 2.7.6 has been released in 7 hours ago.
ref: https://github.com/oneclick/rubyinstaller2/releases

File: td-agent/Rakefile [hunk 1]
```diff
@@ -758,7 +758,7 @@ class BuildTask
     cd(ruby_source_dir) do
       apply_ruby_patches
       sh("./configure", *configure_opts)
-      sh("make", "install", "-j#{Etc.nprocessors}", "DESTDIR=#{STAGING_DIR}")
+      sh("make", "install", "-j1", "DESTDIR=#{STAGING_DIR}")
```
Comment (ashie) [inline td-agent/Rakefile:761]: Oops, it's a mistake.

Comment (ashie) [reply td-agent/Rakefile:761]: hoge! f2c0ff80e98559c48dac580e14d5f457cd7dad9d

**Comment by kenhys** [comment]
could you add a mention to https://github.com/fluent/fluent-package-builder/issues/371?

**Pull request merged**